tex/introduction.tex

```diff
@@ -15,7 +15,54 @@ \section{Introduction}
 
 \subsection{User requirements}
 
-(Might be formulated as user stories, both covering present and desired future workflows.)
+From a user perspective, notebooks may be attractive because they allow for combining a narrative with technical details, code, data and figures and hence provide an integrated way of presenting a line of scientific reasoning, and because they can be created in an incremental and interactive process that in some cases is close the the thought process during scientific work.
+
+Jupyter may be attractive for users because though originating from the Python ecosystem, it provides a common interface for creating notebooks that can contain code in many different languages.  Work with remote Jupyter instances, e.g., through a JupyterHub is popular, because the browser-based graphical application is executed on the user's computer and the bandwidth and stability requirements for the connection to the server are minimal and result in a low-latency and smooth user experience that is rare among classical GUIs that allow for execution on remote servers.
+
+Besides Jupyter, there are other popular implementations of computational notebooks~\citep{Hinsen2019a} that increasingly provide a browser-based way of working with them\footnote{See, e.g., the Matlab Live Editor at https://de.mathworks.com/products/matlab/live-editor.html.}.
```

Comment:
These are very useful aspects that should be stated explicitely, but I would not do so in the user requirements section. I think, we should move this to a place before the user requirements, and let the user requirements section only focus on defining what we, as scientific target users, expect to find in a ready-to-use and highly customizable Jupyter@HPC service. (While moving these paragraphs, could you implement a new line for each sentence?)

Reply:
Moved to main intro for now.